uber/models/attendee.py

```diff
@@ -1111,7 +1131,7 @@ def food_restrictions_filled_out(self):
     @property
     def shift_prereqs_complete(self):
         return not self.placeholder and \
-            self.food_restrictions_filled_out and self.shirt_size_marked
+            self.food_restrictions_filled_out and self.shirt_info_marked
```

Comment:
Note that instead of adding a NEW step to the volunteer checklist, I just expanded the existing "shirt size" step to now include both the size and shirt allotment.  It seemed silly to have two different steps which both asked for shirt information.

This does technically mean that for some people a step which was previously complete will suddenly be un-checked.  However, since we're also sending an email to everyone affected by this explaining the new data they now need to enter, I don't think it'll be an issue, and I believe it's still better this way.